tests/test_data/message/thread_misc.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
There is a lot of additional replication here, isn't there nothing else in the code that you can't reuse/extend?

Reply:
These have been removed in favour of updating the 'category' in thread_json directly.